pkg/cmd/stack/translate_test.go

```diff
@@ -466,7 +466,6 @@ func Test_translateJob(t *testing.T) {
 				CapDrop:       []apiv1.Capability{apiv1.Capability("CAP_DROP")},
 				RestartPolicy: apiv1.RestartPolicyNever,
 				BackOffLimit:  5,
-				Volumes:       []model.StackVolume{{RemotePath: "/volume1"}, {RemotePath: "/volume2"}},
```

Comment:
can we add a test for the two variants? (init container with and without volume)